gnosis/eth/ethereum_client.py

```diff
@@ -634,6 +653,16 @@ def get_slow_provider(self, timeout: int):
         else:
             return self.w3_provider
 
+    def get_network_name(self):
```

Comment:
I would return the enum instead of the string.

Please add the type info `def get_network_name(self) -> EthereumNetworkName:`